includes/today-feeds.php

```diff
@@ -214,3 +214,19 @@ function tu_add_tax_query_to_posts_endpoint( $args, $request ) {
 }
 
 add_action( 'rest_post_query', 'tu_add_tax_query_to_posts_endpoint', 2, 10 );
+
+
+/**
+ * Adds a feed to support the UCF Mobile App concise
+ *
+ * @since 1.2.0
+ * @author RJ Bruneel
+ */
+function add_concise_feed() {
```

Comment:
Can we update the function names for `add_concise_feed()` and `render_concise_feed()` to be prefixed for consistency with the plugin's other functions?   e.g. `tu_add_concise_feed()` and `tu_render_concise_feed()`